react-graphql-apollo-tutorial.md

```diff
@@ -371,45 +371,45 @@ It's up to you to come up with an own folder/file structure for your components
-  * create the CSS *style.css* files in their specified folders from the [appendix](#appendix-styling) section of this tutorial
-  * create the *index.js* files for the components
-  * create further folders on your own for non top level components (e.g. Navigation) when conducting the following sections
-* run the application with `npm start`
- * make sure there are no errors
- * render only a basic App component with *src/App/index.js* in the *src/index.js* file
-* invest 3 minutes of your time and take the {{% a_blank "quiz" "https://www.surveymonkey.com/r/5N9W2WR" %}}
+* If you are not familiar with React, read up *The Road to learn React*
+* Set up the recommended folder/file structure (if you are not going with your own structure and didn't clone the repository)
+  * Create the CSS *style.css* files in their specified folders from the [appendix](#appendix-styling) section of this tutorial
+  * Create the *index.js* files for the components
+  * Create further folders on your own for non top level components (e.g. Navigation) when conducting the following sections
+* Run the application with `npm start`
+ * Make sure there are no errors
+ * Render only a basic App component with *src/App/index.js* in the *src/index.js* file
+* Invest 3 minutes of your time and take the {{% a_blank "quiz" "https://www.surveymonkey.com/r/5N9W2WR" %}}
 
 {{% chapter_header "Configure Apollo Client for React and GitHub's GraphQL API" "react-apollo-client-configuration" %}}
 
-In this section, you will set up a Apollo Client instance as you have done before in the Node.js environment. However, this time you will use Apollo Client directly without the zero-configuration package Apollo Boost. Hence you have to configure the Apollo Client yourself and don't get sensible defaults. Whereas I often believe it's best to use a tool with sensible defaults for the purpose of learning about it, I feel like configuring Apollo yourself will lay out the power of the fine-grained composable ecosystem of Apollo Client to you, how to use them for an initial setup and how to advance your setup with them later on. If you don't understand everything in this section, don't worry too much about it. You can always revisit it later. The thing which matters the most in this section is that you are creating a Apollo Client instance with your own configuration.
+In this section, you will set up a Apollo Client instance in the Node.js environment, same as before. However, this time you will use Apollo Client directly without the zero-configuration package Apollo Boost, meaning you'll need to configure the Apollo Client yourself without sensible defaults. While it's best to use a tool with sensible defaults for learning, configuring Apollo yourself exposes the composable ecosystem of Apollo Client, how to use it for an initial setup, and how to advance this setup later. 
```

Comment:
"In this section, you will set up a Apollo Client instance in the Node.js environment, same as before." -> "In this section, you will set up a Apollo Client instance same as before." <- Maybe my sentence wasn't clear here. It doesn't happen in a Node.js environment this time.